download.md

```diff
@@ -35,6 +35,8 @@ You can run queries using the interactive
 
 Connect to Presto from Java using the
 [JDBC Driver](docs/current/installation/jdbc.html).
+The driver is available in
+[Maven Central](https://search.maven.org/artifact/io.prestosql/presto-jdbc/{{ site.presto_version }}/jar).
```

Comment:
I thought about having something like 
```
<dependency>
  <groupId>io.prestosql</groupId>
  <artifactId>presto-jdbc</artifactId>
  <version>327</version>
</dependency>
```

directly on the page

Reply:
i know the link target (https://search.maven.org/artifact/io.prestosql/presto-jdbc/327/jar) has this, but XML snippet is eye-catching (when someone is looking for it), whereas link is easy to overlook.

Reply:
maybe we add this as a second button, with an icon like https://fontawesome.com/icons/cloud-download-alt?style=solid?

Reply:
Most users just need to download the driver jar file to use in an application like SQuirreL SQL, DBeaver or Superset or so .. not such much write their own app using the driver.

Reply:
but adding it in addition is probably good

Reply:
The XML made the column too wide and messed up the others (caused the button to wrap). Maybe a smaller font would work?

Reply:
> Most users just need to download the driver jar

@mosabua those with SQuirreL -- yes. DBeaver does this for you.
Those building an app based on Presto need to download a jar. (Agreed, this is likely less than 50%)

> Maybe a smaller font would work?

Significantly smaller font may look weird. What about adding a  button (https://github.com/prestosql/prestosql.io/pull/65#discussion_r363628303)?

Reply:
I'm going to merge this, since this is an improvement over what we have now. We can follow up on the various improvement ideas.